mongo/collection.go

```diff
@@ -936,7 +1029,13 @@ func (coll *Collection) Distinct(ctx context.Context, fieldName string, filter i
 	return retArray, replaceErrors(err)
 }
 
-// Find finds the documents matching a model.
+// Find executes a find operation (https://docs.mongodb.com/manual/reference/command/find/) and returns a Cursor over
+// a subset of the documents in the collection.
```

Comment:
We typically don't talk about "sets" in MongoDB docs.  Instead , something like "over the matching documents in the collection".

Reply:
done.